src/index.js

```diff
@@ -16,6 +16,7 @@ class Dropzone extends React.Component {
     this.onDragLeave = this.onDragLeave.bind(this);
     this.onDragOver = this.onDragOver.bind(this);
     this.onDrop = this.onDrop.bind(this);
+    this.handleOnCancel = this.handleOnCancel.bind(this);
```

Comment:
The naming is inconsistent with the rest of methods. Also, I can't figure out from the name what's for native event it represents.